OpenSSL/crypto.py

```diff
@@ -1520,11 +1519,15 @@ class X509StoreContext(object):
     An X.509 store context.
 
     An :py:class:`X509StoreContext` is used to define some of the criteria for
-    certificate verification.  The information encapsulated in this object
+    certificate verification. The information encapsulated in this object
```

Comment:
Haha, nice @hynek. iOS retrained me since double space turns into a single space and a period. Prior to 2007 I was a double space after period loyalist though!